tests/admin_views/forms.py

```diff
@@ -14,6 +14,18 @@ def clean_username(self):
         return username
 
 
+class CustomAdminPasswordChangeForm(AdminPasswordChangeForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["old_password"].widget.attrs.update({"autocomplete": "off"})
+
+    def clean_new_password1(self):
+        new_password1 = self.cleaned_data.get("new_password1")
+        if new_password1 == "forbidden_password":
+            raise ValidationError("forbidden password form error")
+        return new_password1
```

Comment:
```suggestion
class CustomAdminPasswordChangeForm(AdminPasswordChangeForm):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.fields["old_password"].label = "Custom old password label"
```
I would simplify this and have 1 test just to prove this form is being rendered:
```python
    def test_custom_password_change_form(self):
        user = User.objects.get(username="super")
        self.client.force_login(user)
        response = self.client.get(reverse("admin4:password_change"))
        self.assertContains(response, "Custom old password label")
```